### ISSUE TITLE
Google Scholar labels not working

### ISSUE THREAD

**Issue by staticdev** [mcmtroffaes/sphinxcontrib-bibtex#118]
Hello,

I tried the minimal example with a Bibtex reference generated by Google Scholar:

@article{srivastava2014dropout,
  title={Dropout: a simple way to prevent neural networks from overfitting.},
  author={Srivastava, Nitish and Hinton, Geoffrey E and Krizhevsky, Alex and Sutskever, Ilya and Salakhutdinov, Ruslan},
  journal={Journal of Machine Learning Research},
  volume={15},
  number={1},
  pages={1929--1958},
  year={2014}
}

And citing like:

:cite:`srivastava2014dropout`

.. bibliography:: refs.bib
  :style: unsrt

And it just ignores the citation. How do I fix it without having to change all my references generated by Google Scholar?

Thanks in advance.

**Comment by mcmtroffaes** [comment]
I've tried to reproduce, and your bibtex code works perfectly fine for me:

https://gist.github.com/mcmtroffaes/f26684d5c7febf6a5698c0fb0467072e

Not sure what else I can suggest. You can try adding :all: to your bibliography directive to ensure that you don't miss any citations, or move your bibliography to a file that comes last in the alphabet:

https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#unresolved-citations-across-documents
